ember-rails.gemspec

```diff
@@ -15,6 +15,7 @@ Gem::Specification.new do |s|
   s.add_dependency "railties", [">= 3.1"]
   s.add_dependency "active_model_serializers"
   s.add_dependency "barber", [">= 0.6.0"]
+  s.add_dependency "es6_module_transpiler-rails"
```

Comment:
we hope to move ti esperanto for the module transpiles soon https://github.com/ef4/broccoli-es6modules/pull/4 at somepoint (when we switch) we should coordinate that this switches aswell.